packages/mediapipe-task-text/example/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
Are all of the analysis options meant to be the same? Trying to keep them form drifting too far apart for consistency.